### PR TITLE
Adding 21-25 of list programs

### PR DESCRIPTION
**Pull request by anilhariparamhans**

### CODE REVIEW

File: GeeksforGeeks/vAnil/03. List Programs/21.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+'''
+
+xlist = [11, 5, 17, 18, 23, 50]
+#yRem = [1:5]
+
+del xlist[1:5]
+
+print(*xlist)
+
+
```
Comment (dimplenaresh1) [inline GeeksforGeeks/vAnil/03. List Programs/21.py:37]: Add the code for removing element by index.

**Review by dimplenaresh1 — CHANGES_REQUESTED** [review]

File: GeeksforGeeks/vAnil/03. List Programs/26.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Output : output_list = [-1]
+'''
+
+list1 = [10, 20, 30, 20, 20, 30, 40, 50, -20, 60, 60, -20, -20]
+
+list2 = [list1[0]]
+
+for item in list1:
+    if item not in list2:
+        list2.append(item);
```
Comment (dimplenaresh1) [inline GeeksforGeeks/vAnil/03. List Programs/26.py]: We have to print the duplicates. Here you are removing them.

**Review by dimplenaresh1 — CHANGES_REQUESTED** [review]

File: GeeksforGeeks/vAnil/03. List Programs/21.py (hunk 1 above)
Comment (anilhariparamhans) [reply GeeksforGeeks/vAnil/03. List Programs/21.py:37]: Incorporated this review comment and updated the code.

File: GeeksforGeeks/vAnil/03. List Programs/26.py (hunk 2 above)
Comment (anilhariparamhans) [reply GeeksforGeeks/vAnil/03. List Programs/26.py]: Yes right. I updated the logic.

**Review by dimplenaresh1 — APPROVED** [review]

**Pull request merged**